### PR TITLE
Reviewer - chars changes highlighting

### PR DESCRIPTION
**Pull request by webdevelopland**
The PR adds additional highlighting of separate chars.

### CODE REVIEW

File: tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts [hunk 1]
```diff
@@ -24,4 +28,83 @@ export class UIService {
...
+        blockLine.clearCode,
+        className,
+      );
+    }
+  }
+
+  // Escape html special chars, and enclose highlighted part in <span> tag
+  // example:
+  // 'I think x > y here' -> 'I <span>think</span> x &gt; y here'
+  // word 'think' is highlighted part
```
Comment (oferb) [inline tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts]: Great documentation.

File: tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
 
-import { BlockIndex, BlockLine, ChangesLine } from '../code-changes.interface';
+import {
+  BlockIndex,
+  BlockLine,
+  ChangesLine,
+} from '../code-changes.interface';
 
 @Injectable()
 export class UIService {
```
Comment (MateMahinKhan) [inline tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts]: maybe change the name for this service to something more related to text and lines. UI would mean that it does something for User Interface.

**Comment by MateMahinKhan** [comment]
highlightjs.scss   shouldn't this file be near it's component. or Are styles in this file used all across the app?

File: tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts (hunk 1 above)
Comment (webdevelopland) [reply tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts]: :)

File: tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts (hunk 2 above)
Comment (webdevelopland) [reply tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts]: Yeah, I was thinking about it too.

Comment (webdevelopland) [reply tools/reviewer/web/src/app/dashboard/file-changes/code-changes/services/ui.service.ts]: > is `highlightjs.scss` used all across the app?
Yes! It's part of gasd. It create span tags while run-time, so the styles can be part of component.

**Pull request merged**